projects/freedesktop.org/libslirp/package.yml

```diff
@@ -0,0 +1,48 @@
+distributable:
+  url: https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v{{ version }}/libslirp-v{{ version }}.tar.gz
```

Comment:
looks like we should name this `freedesktop.org/slirp` unless there's a project by that name. maybe. I tend to prefer the undecorated name, but we have both.